ipam/status.go

```diff
@@ -36,8 +36,10 @@ func NewStatus(allocator *Allocator, defaultSubnet address.CIDR) *Status {
 	}
 
 	var paxosStatus *paxos.Status
-	if allocator.paxosActive {
-		paxosStatus = paxos.NewStatus(allocator.paxos)
+	if allocator.awaitingConsensus && allocator.participant != nil {
+		if node, ok := allocator.participant.(*paxos.Node); ok {
+			paxosStatus = paxos.NewStatus(node)
+		}
```

Comment:
That's a bit ugly. And I would have thought we probably want to show something in the status for non-electors.